### ISSUE TITLE
Read stream never ends on v3.0.0

### ISSUE THREAD

**Issue by claustres** [jb55/s3-blob-store#17]
It seems that the stream is not correctly ended on the latest version. This code https://github.com/feathersjs-ecosystem/feathers-blob/blob/master/src/index.js#L37 previously worked (v2.1.0) and now the promise is never resolved.

**Comment by jb55** [comment]
ok I should probably get around to fixing the test suite, thanks for letting me know. A minimal failing test case would be useful to fix this.

**Comment by claustres** [comment]
I think the test suite already fails because of this: `test exited without ending`.

**Comment by claustres** [comment]
I simply tested something like this I think:
```
store.createReadStream(...)
.pipe(fs.createWriteStream('test.txt'))
.on('finish', callback)
```
The file was correctly produced but the event never raised.

**Comment by jb55** [comment]
On it. Fixing.

**Comment by jb55** [comment]
Fixed in 3.1.0, reverted change in 3.1.0 for now
